### DISCUSSION TITLE
Make __vanish__ part of the public API

### DISCUSSION THREAD

**Discussion by bradgessler** [yippee-fun/phlex#583] [Ideas]
Sometimes a component needs to be rendered, but the output is not needed, like in this controller at https://github.com/rubymonolith/classic/blob/47d9fee6e17f5f49d7dfbefee4b194b6eddcff8d/app/controllers/posts_controller.rb#L71-L73 so Superform can permit params based on the contents of a form.

Could this [`__vanish__` API](https://github.com//phlex-ruby/phlex/blob/9e5820dc69cb243e31b924f00390deb45be18fb3/lib/phlex/sgml.rb#L238-L248) be made available for such calls? I see I can include `DeferredRender` to enable that, but I only need this for permitting params. When I'm actually rendering the form in a view I don't need to defer rendering.

**Comment by joeldrapper** [comment]
I think this makes sense. 👍
